### PR TITLE
Updated node.md

### PR DESCRIPTION
**Pull request by erictune**
Describe kubelet self-registration.
Remove references to deprecated node controller flags.
Describe flags that control kubelet registration.

### CODE REVIEW

**Comment by erictune** [comment]
continuing #8821

File: docs/node.md [hunk 1]
```diff
@@ -99,25 +99,37 @@ Node Controller is unable to provision the node for you, i.e. it won't install
...
-Note if Kubernetes is running on cloud provider, `kubectl create` a node will
-be refused if Node Controller has already synchronized nodes from cloud provider.
-Admin can choose to make the node unschedulable using `kubectl`. Unscheduling the node
-will not affect any existing pods on the node but it will disable creation of
-any new pods on the node. Node unschedulable example:
+services. 
+
+### Self-Registration of nodes
+
+When kubelet flag is `--register-node`,  is true (the default), then the kubelet will attempt to
```
Comment (cjcullen) [inline docs/node.md]: When kubelet flag `--register-node` is true (remove the is and the trailing comma).

File: docs/node.md [hunk 2]
```diff
@@ -99,25 +99,37 @@ Node Controller is unable to provision the node for you, i.e. it won't install
...
+  - `--register-node` tells the kubelet to create its own node resource.
+
+Currently, any kubelet is authorized to create/modify any node resource, but in practice it only creates/modifies
+its own.  (In the future, we plan to limit authorization to only allow a kubelet to modify its own Node resource.)
+
+#### Manual Node Administration
+
+A cluster administrator can create and modify Node objects.
+
+If the administrator wishes to create node objects manually, set kubelet flag `--register-node=0`.
```
Comment (cjcullen) [inline docs/node.md]: Does false work for this flag? That would make more sense to me, but I'll leave that up to you.

File: docs/node.md [hunk 3]
```diff
@@ -99,25 +99,37 @@ Node Controller is unable to provision the node for you, i.e. it won't install
...
+
+The administrator can modify Node resources (regardless of the setting of `--register-node`).
+Modifications include setting labels on the Node, and marking it unschedulable.
+
+Labels on node can be used in conjuction with node selectors on pods to control scheduling.
```
Comment (cjcullen) [inline docs/node.md]: Labels on nodes

**Comment by cjcullen** [comment]
LGTM after a few nits.

**Comment by kelseyhightower** [comment]
LGTM

File: docs/node.md (hunk 2 above)
Comment (erictune) [inline docs/node.md]: fixed.

File: docs/node.md (hunk 3 above)
Comment (erictune) [inline docs/node.md]: fixed.

**Comment by erictune** [comment]
ptal

**Comment by erictune** [comment]
Fixes #6087 by documenting it.

**Comment by cjcullen** [comment]
Have you pushed changes?

**Comment by erictune** [comment]
whoops.  pushed now.

**Comment by cjcullen** [comment]
LGTM. Shippable is failing because gendocs needs to be run for that new admission_controllers doc.

**Pull request merged**